src/lib/types/domain/rehearsal.ts

```diff
@@ -2,15 +2,18 @@ import type { DocumentReference, Timestamp } from 'firebase/firestore'
 
 export type rehearsal = {
 	id: string
-	edition: DocumentReference // Reference to edition
-	startTime: Timestamp
-	endTime: Timestamp
-	location: string
 	songsToRehearse: {
 		song: DocumentReference // Reference to song
 		startTime: Timestamp
 		endTime: Timestamp
 	}
```

Comment:
Wil je deze niet ook bij newRehearsal? Anders heb je misschien een probleem als je de data ophaalt. Also, dit moet een array zijn

Reply:
newRehearsal is alleen als ie aangemaakt wordt, en dat is zonder nummers dus denk t niet?